### PR TITLE
add parse crash reporting

### PR DESCRIPTION
**Pull request by tongxiang**
#### What's this PR do?
Adds Parse crash reporting. 

#### How should this be manually tested?
We've added some exception code (below) to the AppDelegate within `didFinishLaunchingWithOptions`. We've run the app on an iPhone running the app connected to a machine, and thrown the test exception below a few times. We're waiting to see whether or not this implementation of Parse crash reporting actually works, and actually shows up in the Parse console. 

```
    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(5.0 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
        [NSException raise:NSGenericException format:@"Everything is ok. This is just a test crash."];
    });
```
#### Any background context you want to provide?
See #60 for a bit more details into what we've went to. 

#### What are the relevant tickets?
#60. 

#### Questions:
Is this actually working with Parse? Do we need to install the Parse CLI onto each dev's machine before the build works? (We will test this with @jonuy).

### CODE REVIEW

**Comment by aaronschachter** [comment]
We'll want to add the instructions to install the Parse CLI to the README if we end up going this route @tongxiang

**Comment by aaronschachter** [comment]
@tongxiang did we ever run the working crash test from our phones? Parse still is not showing any crash reports, but perhaps we only ran from iOS simulator...

**Comment by tongxiang** [comment]
1. FYI @aaronschachter we tested on Uy's machine and parse symbol uploading was possible even without the CLI installed on his machine. But it seems reasonable to include that in the readme. 

2. And yes, we did run the working crash test from our phone after connecting it to our machines and downloading from XCdoe.

**Comment by aaronschachter** [comment]
Ran from my device this morning, still waiting for a crash report to show in Parse (about 5 hours later).

**Comment by aaronschachter** [comment]
Still no crash reports displayed in Parse :snail: :crying_cat_face:

File: parse/config/global.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "global": {
+    "parseVersion": "1.4.2"
+  },
+  "applications": {
+    "Let's Do This": {
+      "applicationId": "kP8eFOvgTZc3EVSA04RroPC6tSEnkhEBr4oYg2qG",
```
Comment (aaronschachter) [inline parse/config/global.json]: Just realizing that this is no good for open source land :(

**Comment by aaronschachter** [comment]
I'm still not seeing any Crash Reports in the Parse admin UI, which is a bummer. I also didn't catch that the `global.json` added stores the Parse ID's, which is also a bummer.

My original thought on this was to merge it, and then take it out later if we can't get it working (or want to use a different platform), but I'm not really into the Parse ID's within our repo. We'll need to scrub this PR to get rid of them... which makes me think it might not be worth the work to refactor the Parse config stuff if we can't even get Crash Reports to show up...

**Comment by aaronschachter** [comment]
Lets close this for now and revisit when we're a little further along with the app.

**Pull request closed without merging**